coordinator/src/routes/tracker/tracker.ts

```diff
@@ -0,0 +1,57 @@
+import Hapi from "@hapi/hapi"
+import {hl7V3, fhir} from "@models"
+import * as LosslessJson from "lossless-json"
+import {BASE_PATH, ContentTypes} from "../util"
+import {getRequestId} from "../../utils/headers"
+import {createBundle} from "../../services/translation/common/response-bundles"
+import {trackerClient} from "../../services/communication/tracker/tracker-client"
+
+/* The PAUI Tracker */
+
+export default [{
+  method: "GET",
+  path: `${BASE_PATH}/Tracker`,
+  handler: async (
+    request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit
+  ): Promise<Hapi.Lifecycle.ReturnValue> => {
+    const requestQuery = {
+      prescription_id: request.query.prescription_id as string,
+      repeat_number: request.query.repeat_number as string
+    }
+
+    const clientResponse = await trackerClient.track(
+      getRequestId(request.headers),
+      requestQuery.prescription_id,
+      requestQuery.repeat_number,
+      request.logger
+    )
+
+    const response = clientResponse.prescription
```

Comment:
have a look at making use of `x-raw-response`?

Reply:
Should be done, will test tomorrow